test/mobile/test_bytecode.py

```diff
@@ -0,0 +1,10 @@
+from torch.jit.mobile import _get_runtime_bytecode_version
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+class testVariousModelVersions(TestCase):
+    def test_get_runtime_bytecode_version(self):
+        runtime_bytecode_version = _get_runtime_bytecode_version()
+        assert(isinstance(runtime_bytecode_version, int))
```

Comment:
maybe also >=0

Reply:
int or int64_t?

Reply:
int64_t is a macro we defined in cpp so we don't really have it in Python. `int` in Python should be sufficient to cover long type from cpp.